### PR TITLE
Link to the SAML troubleshooting doc from guide

### PR DESCRIPTION
**Pull request by jkakavas**
Explicitly call out the existence of the troubleshooting guide so
that hopefully users can solve common and easy problems with their
initial configuration

### CODE REVIEW

File: x-pack/docs/en/security/authentication/saml-guide.asciidoc [hunk 1]
```diff
@@ -858,3 +858,15 @@ xpack.security.authc.realms.saml.saml_eng:
 It is possible to have one or more {kib} instances that use SAML, while other
 instances use basic authentication against another realm type (e.g.
 <<native-realm, Native>> or <<ldap-realm, LDAP>>).
+
+=== Troubleshooting SAML Realm Configuration
+
+The SAML 2.0 specification offers a lot of options and flexibility for the implementers
+of the standard which in turn adds to the complexity and the number of configuration options
+that is available both at the Service Provider (Elastic Stack) and at the Identity Provider.
```
Comment (lcawl) [inline x-pack/docs/en/security/authentication/saml-guide.asciidoc]:
```suggestion
that are available both at the Service Provider (Elastic Stack) and at the Identity Provider.
```

**Review by lcawl — APPROVED** [review]
LGTM

File: x-pack/docs/en/security/authentication/saml-guide.asciidoc [hunk 2]
```diff
@@ -858,3 +858,15 @@ xpack.security.authc.realms.saml.saml_eng:
...
+=== Troubleshooting SAML Realm Configuration
+
+The SAML 2.0 specification offers a lot of options and flexibility for the implementers
+of the standard which in turn adds to the complexity and the number of configuration options
+that are available both at the Service Provider (Elastic Stack) and at the Identity Provider.
+Additionally, different security domains have different security requirements that need
+specific configuration to be satisfied.
+A conscious effort has been made to mask this complexity with sane defaults and the detailed
+documentation above but in case you encounter issues while configuring a SAML realm, you can
+look through our {stack-ov}/saml-guide.html[SAML troubleshooting documentation] that has
```
Comment (tvernum) [inline x-pack/docs/en/security/authentication/saml-guide.asciidoc]: Is that the right link? Don't you want to link to `trb-security-saml.html`?

**Review by tvernum — APPROVED** [review]
LGTM, except I think the link is wrong.

File: x-pack/docs/en/security/authentication/saml-guide.asciidoc (hunk 2 above)
Comment (jkakavas) [reply x-pack/docs/en/security/authentication/saml-guide.asciidoc]: doh !yeah, an infinite doc loop is not what I had in mind :) thanks for noticing

Comment (lcawl) [reply x-pack/docs/en/security/authentication/saml-guide.asciidoc]: Good catch!

File: x-pack/docs/en/security/authentication/saml-guide.asciidoc [hunk 3]
```diff
@@ -858,3 +858,15 @@ xpack.security.authc.realms.saml.saml_eng:
...
+=== Troubleshooting SAML Realm Configuration
+
+The SAML 2.0 specification offers a lot of options and flexibility for the implementers
+of the standard which in turn adds to the complexity and the number of configuration options
+that are available both at the Service Provider (Elastic Stack) and at the Identity Provider.
+Additionally, different security domains have different security requirements that need
+specific configuration to be satisfied.
+A conscious effort has been made to mask this complexity with sane defaults and the detailed
+documentation above but in case you encounter issues while configuring a SAML realm, you can
+look through our {stack-ov}/trb-security-saml[SAML troubleshooting documentation] that has
```
Comment (lcawl) [inline x-pack/docs/en/security/authentication/saml-guide.asciidoc]:
```suggestion
look through our {stack-ov}/trb-security-saml.html[SAML troubleshooting documentation] that has
```

Comment (jkakavas) [reply x-pack/docs/en/security/authentication/saml-guide.asciidoc]: Let's see in how many different ways I can mess this simple PR up :)

**Comment by jkakavas** [comment]
@elasticmachine run the default distro tests σε παρακαλώ

**Pull request merged**